_ont/ont-zyxel-pmg3000-d20b.md

```diff
@@ -55,6 +55,8 @@ The stick has a TTL 3.3v UART console (configured as 115200 8-N-1) that can be a
 ## List of software versions
 - V1.00(ABVJ.0)b3s (2020)
 - V1.00(ABVJ.0)b3i (2020)
+- V1.00(ABVJ.0)b3v
+- ```
```

Comment:
ummh what

Reply:
Yes, this is fixed, I also seen it in the preview :D